src/CSSmodule/details.css

```diff
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.detailsection {
+  margin-top: 5rem;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  background-color: black;
+}
+
+.detailsection:nth-child(even) {
+  background-color: darkgrey;
+}
+
+.imgDt {
+  width: 15rem;
+}
+
+.detailPage {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-start;
+  list-style: none;
+  width: 100%;
+  padding: 10px;
+}
+
+.detailPage li {
+  border: 1px solid black;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  color: #fff;
+}
+
+.coinsIcon {
+  background-color: rgb(49, 49, 49);
+}
+
+body {
+  background-color: black;
+}
+
+.green {
+  color: rgb(30, 238, 30);
+}
+
+.red {
+  color: red;
+}
```

Comment:
- [ ] `Required`
- [ ] as the previous `reviewer` mentioned, according to the requirements the Design
You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.

- [ ] please see screenshots and please make adjustments 🥂 

`your design`
![image](https://user-images.githubusercontent.com/66230306/219815069-b163ea5e-ea38-40ba-93c8-0a6030a5c0d4.png)

`origin design`
![image](https://user-images.githubusercontent.com/66230306/219815193-56fe0a47-a62b-4106-9e77-7aa9a4b7152a.png)

Reply:
Implemented, Thank you very much.